### PR TITLE
fix: remove unwanted newline which caused the paragraph to break

### PR DESCRIPTION
**Pull request by akmalick**
Removed newline after "mentorship, and sponsorship to" to fix broken paragraph.

### CODE REVIEW

File: locale/en/index.md [hunk 1]
```diff
@@ -163,12 +163,7 @@ We have an [orientation guide](https://github.com/openjs-foundation/summit/blob/
 to help you figure out how you might want to take part. If you have any questions or
 want to talk about anything, please reach out to [blacklivesmatter@nodejs.org](mailto:blacklivesmatter@nodejs.org).
 
-Moving forward we plan to offer ongoing support, mentorship, and sponsorship to
-
-help those from the Black community get involved in our project. We will
-aslo be analyzing our existing processes and governance to see what changes
-we can make to make sure our project is the kind of environment that the
-Black community wants to participate in.
+Moving forward we plan to offer ongoing support, mentorship, and sponsorship to help those from the Black community get involved in our project. We will aslo be analyzing our existing processes and governance to see what changes we can make to make sure our project is the kind of environment that the Black community wants to participate in.
```
Comment (MylesBorins) [inline locale/en/index.md]:
```suggestion
Moving forward we plan to offer ongoing support, mentorship, and sponsorship to
help those from the Black community get involved in our project.
also be analyzing our existing processes and governance to see what changes
we can make to make sure our project is the kind of environment that the
Black community wants to participate in.
```

**Review by MylesBorins — COMMENTED** [review]
I had just opened https://github.com/nodejs/nodejs.org/pull/3255/ to fix this exact same issue (extra newline) as well as fix the spelling error s/aslo/also

I've added the changes i've made here as a suggested change so we could land this PR instead. We had updated formatting through the entire doc to make paragraphs multiple lines, so it would be odd to change that formatting for just this section

File: locale/en/index.md [hunk 2]
```diff
@@ -164,9 +164,8 @@ to help you figure out how you might want to take part. If you have any question
 want to talk about anything, please reach out to [blacklivesmatter@nodejs.org](mailto:blacklivesmatter@nodejs.org).
 
 Moving forward we plan to offer ongoing support, mentorship, and sponsorship to
-
-help those from the Black community get involved in our project. We will
-aslo be analyzing our existing processes and governance to see what changes
+help those from the Black community get involved in our project.
```
Comment (MylesBorins) [inline locale/en/index.md]:
```suggestion
help those from the Black community get involved in our project. We will
```

**Review by MylesBorins — APPROVED** [review]
LGTM

**Comment by akmalick** [comment]
> I had just opened #3255 to fix this exact same issue (extra newline) as well as fix the spelling error s/aslo/also
> 
> I've added the changes i've made here as a suggested change so we could land this PR instead. We had updated formatting through the entire doc to make paragraphs multiple lines, so it would be odd to change that formatting for just this section

Appreciate that ❤️   @MylesBorins  
and a great initiative by Nodejs Team. My love and prayers for all of you.

**Pull request merged**